api/envoy/data/accesslog/v2/accesslog.proto

```diff
@@ -185,12 +185,14 @@ message ResponseFlags {
   // Indicates if the request was deemed unauthorized and the reason for it.
   Unauthorized unauthorized_details = 13;
 
-  // [#not-implemented-hide:] Hide from docs.
```

Comment:
Unrelated clean up?

Reply:
I discovered that it was in fact implemented already, so I figured I'd fix the docs while I was touching everything:

https://github.com/envoyproxy/envoy/blob/b71bd22c99d36037e317a65260eba75a91cdafd7/source/extensions/access_loggers/http_grpc/grpc_access_log_impl.cc#L148